client/components/Welcome.jsx

```diff
@@ -1,9 +1,14 @@
 import React from 'react';
+import {BrowserRouter, Route} from 'react-router-dom'
 import styled from 'styled-components';
 import { MenuCards } from 'Components/MenuCards';
 import { Navigation } from 'Components/Navigation';
 import SplashImage from 'Assets/Red-Lotus';
 import { GlobalStyles } from 'Styles/globalStyles';
+import { GREY, DARK_BLUE, LIGHT_BLUE, WHITE, RED, YELLOW } from 'Styles/colors';
+import { Switch } from 'react-router-dom';
+import { About } from './About';
+import { DownloadApp } from './DownloadApp';
 import { GREY, DARK_BLUE, WHITE, RED } from 'Styles/colors';
```

Comment:
You are importing the color variables twice which is throwing an error. Please only import once. Might have been caused by VScode trying to auto import.